xml/System.Threading.Tasks.Dataflow/BatchedJoinBlock`2.xml

```diff
@@ -112,6 +112,7 @@
         <summary>Initializes a new <see cref="T:System.Threading.Tasks.Dataflow.BatchedJoinBlock`2" /> with the specified configuration.</summary>
         <remarks>To be added.</remarks>
         <exception cref="T:System.ArgumentNullException">The <paramref name="dataflowBlockOptions" /> is <see langword="null" />.</exception>
```

Comment:
We either say `The <paramref name="dataflowBlockOptions" /> parameter is null` or `<paramref name="dataflowBlockOptions" /> is null`

```suggestion
         <exception cref="T:System.ArgumentNullException"><paramref name="dataflowBlockOptions" /> is <see langword="null" />.</exception>
```